### ISSUE TITLE
Error in SublimeLinter daemon

### ISSUE THREAD

**Issue by darkred** [SublimeLinter/SublimeLinter3#404]
Using ST3 build 3095 x64, with `Sublimelinter` 3.6.1 and `Sublimelinter-eslint` 1.9.2 (and `eslint 1.10.3`, in win 10 x64).

All of a sudden today, whatever javascript file I open, is not linted anymore.
Instead this is what I get:

```
DPI scale: 1
startup, version: 3095 windows x64 channel: dev
executable: /C/Program Files/Sublime Text 3/sublime_text.exe
working dir: /C/Users/Kostas/AppData/Roaming/Mozilla/Firefox/Profiles/xxxxxxx.default/gm_scripts/4chan_-_filter_highlighting_fix
packages path: /C/Users/Kostas/AppData/Roaming/Sublime Text 3/Packages
state path: /C/Users/Kostas/AppData/Roaming/Sublime Text 3/Local
zip path: /C/Program Files/Sublime Text 3/Packages
zip path: /C/Users/Kostas/AppData/Roaming/Sublime Text 3/Installed Packages
ignored_packages: ["BracketHighlighter", "ConvertToUTF8", "DocBlockr", "Git", "HTML-CSS-JS Prettify", "jQuery", "PackageResourceViewer", "SublimeLinter-jshint", "TrailingSpaces", "Vintage"]
pre session restore time: 0.056839
startup time: 0.075839
first paint time: 0.077839
reloading plugin Default.block
reloading plugin Default.comment
reloading plugin Default.convert_syntax
reloading plugin Default.copy_path
reloading plugin Default.delete_word
reloading plugin Default.detect_indentation
reloading plugin Default.duplicate_line
reloading plugin Default.echo
reloading plugin Default.exec
reloading plugin Default.fold
reloading plugin Default.font
reloading plugin Default.goto_line
reloading plugin Default.history_list
reloading plugin Default.indentation
reloading plugin Default.kill_ring
reloading plugin Default.mark
reloading plugin Default.new_templates
reloading plugin Default.open_context_url
reloading plugin Default.open_file_settings
reloading plugin Default.open_in_browser
reloading plugin Default.pane
reloading plugin Default.paragraph
reloading plugin Default.paste_from_history
reloading plugin Default.quick_panel
reloading plugin Default.run_all_syntax_tests
reloading plugin Default.save_on_focus_lost
reloading plugin Default.scroll
reloading plugin Default.set_unsaved_view_name
reloading plugin Default.side_bar
reloading plugin Default.sort
reloading plugin Default.swap_line
reloading plugin Default.switch_file
reloading plugin Default.symbol
reloading plugin Default.transform
reloading plugin Default.transpose
reloading plugin Default.trim_trailing_white_space
reloading plugin CSS.css_completions
reloading plugin Diff.diff
reloading plugin HTML.encode_html_entities
reloading plugin HTML.html_completions
reloading plugin 0_package_control_loader.00-package_control
reloading plugin 0_package_control_loader.02-bz2
reloading plugin 0_package_control_loader.50-backrefs
reloading plugin Package Control.1_reloader
reloading plugin Package Control.2_bootstrap
reloading plugin Package Control.Package Control
reloading plugin SublimeLinter.commands
reloading plugin SublimeLinter.sublimelinter
reloading plugin SublimeLinter-contrib-eslint.linter
SublimeLinter: eslint linter loaded 
plugins loaded
SublimeLinter: debug mode: off 
SublimeLinter: error in SublimeLinter daemon: 
SublimeLinter: -------------------- 
SublimeLinter: Traceback (most recent call last):
  File "C:\Users\Kostas\AppData\Roaming\Sublime Text 3\Packages\SublimeLinter\lint\queue.py", line 57, in loop
    item = self.q.get(block=True, timeout=self.MIN_DELAY)
  File "./queue.py", line 175, in get
queue.Empty

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Kostas\AppData\Roaming\Sublime Text 3\Packages\SublimeLinter\lint\queue.py", line 65, in loop
    self.lint(view_id, timestamp)
  File "C:\Users\Kostas\AppData\Roaming\Sublime Text 3\Packages\SublimeLinter\lint\queue.py", line 111, in lint
    self.callback(view_id, timestamp)
  File "C:\Users\Kostas\AppData\Roaming\Sublime Text 3\Packages\SublimeLinter\sublimelinter.py", line 119, in lint
    Linter.lint_view(view, filename, code, hit_time, callback)
  File "C:\Users\Kostas\AppData\Roaming\Sublime Text 3\Packages\SublimeLinter\lint\linter.py", line 924, in lint_view
    linter.lint(hit_time)
  File "C:\Users\Kostas\AppData\Roaming\Sublime Text 3\Packages\SublimeLinter\lint\node_linter.py", line 77, in lint
    super(NodeLinter, self).lint(hit_time)
  File "C:\Users\Kostas\AppData\Roaming\Sublime Text 3\Packages\SublimeLinter\lint\linter.py", line 1384, in lint
    output = self.run(cmd, self.code)
  File "C:\Users\Kostas\AppData\Roaming\Sublime Text 3\Packages\SublimeLinter\lint\linter.py", line 1790, in run
    return self.communicate(cmd, code)
  File "C:\Users\Kostas\AppData\Roaming\Sublime Text 3\Packages\SublimeLinter-contrib-eslint\linter.py", line 79, in communicate
    if '__RELATIVE_TO_FOLDER__' in cmd and self.filename[0] == self.view.window().extract_variables()['folder'][0]:
KeyError: 'folder'
 
SublimeLinter: -------------------- 
Package Control: Skipping automatic upgrade, last run at 2016-01-31 00:23:59, next run at 2016-01-31 01:23:59 or after

```
I normally have all packages installed (see the `ignored_packages` line) but I currently disabled all packages but `SublimeLinter` and `SublimeLinter-eslint`.

FWIW what I only tried today was to install the `HexViewer` package.
After some time I opened a .js file, and I noticed the issue.
Then I tried disabling or removing `HexViewer`,
and even tried removing+re-installing `SublimeLinter` and `SublimeLinter-eslint`, but the issue remains.

<br>
PS. On the other hand, if I disable `SublimeLinter-eslint` and enable `SublimeLinter-jshint`, then linting (via jshint) works ok.

**Comment by groteworld** [comment]
This would be better reported to the specific linter [SublimeLinter-eslint](https://github.com/roadhump/SublimeLinter-eslint/) but it seems to be a problem possibly from https://github.com/roadhump/SublimeLinter-eslint/commit/2689471e45d5c1b6f4cd8bc060e33c0b6711940d

cc @roadhump

**Comment by roadhump** [comment]
@darkred Should be fixed in SublimeLinter-eslint v1.9.3

**Comment by darkred** [comment]
@roadhump Yes, it's fixed. Thank you very much.
